appservice/src/tree/DeploymentsTreeItem.ts

```diff
@@ -14,22 +14,27 @@ import { ScmType } from '../ScmType';
 import { retryKuduCall } from '../utils/kuduUtils';
 import { DeploymentTreeItem } from './DeploymentTreeItem';
 import { getThemedIconPath } from './IconPath';
+import { ISiteTreeRoot } from './ISiteTreeRoot';
 
 /**
  * NOTE: This leverages a command with id `ext.prefix + '.connectToGitHub'` that should be registered by each extension
  */
 export class DeploymentsTreeItem extends AzExtParentTreeItem {
     public static contextValueConnected: string = 'deploymentsConnected';
     public static contextValueUnconnected: string = 'deploymentsUnconnected';
+    public static contextValueTrialApp: string = 'deploymentsTrialApp';
```

Comment:
I would keep trial app stuff out of this repo if possible. For example, you could just create a class in the app service repo that extends DeploymentsTreeItem and overwrites the contextValue